src/main/resources/assets/paths/darV2.yaml

```diff
@@ -1,6 +1,9 @@
 post:
-  summary: Create Data Access Request, version 2
-  description: Creates a Data Access Request
+  deprecated: true
```

Comment:
Again, did we want to deprecate this now and not wait for the cleanup to completely switch over to the `v3` endpoint?